### PR TITLE
Browser source merge

### PR DESCRIPTION
**Pull request by Y0dax**
~~@DomNomNom Would you be able to take a look at this and attempt to get your emote support working in app.tsx which serves as the new static display for local express hosting and OBS browser source capture.~~

Got the new emote support working!!! This is a merge of the express server branch and master. @alexgQQ

### CODE REVIEW

**Comment by Y0dax** [comment]
@alexgQQ this includes a manual merge of the new app tsx and the old OBS component so that the new functionality works correctly in the browser.

File: src/main.prod.js.LICENSE.txt [hunk 1]
```diff
@@ -1 +1,365 @@
+/*
```
Comment (alexgQQ) [inline src/main.prod.js.LICENSE.txt:1]: I don't think this needs to be committed.

**Comment by alexgQQ** [comment]
@Y0dax Fantastic! I did update my original PR for the merge, but if this keeps that gif support, lets use it 👍

**Comment by alexgQQ** [comment]
> @Y0dax Fantastic! I did update my original PR for the merge, but if this keeps that emote support, lets use it 👍

Actually scratch that, checked this before discord. I merged this into the original branch, seems to all work 👍

**Pull request merged**